apple/internal/partials/framework_import.bzl

```diff
@@ -306,6 +310,7 @@ def _generate_symbols(
 def framework_import_partial(
         *,
         actions,
+        features = [],
```

Comment:
I don’t think this should have a default value here but this reduces the patch amount a bit, as we don’t have a need for this feature for non-iOS platforms.